### ISSUE TITLE
HttpFactory keeps adding 'Accept' headers

### ISSUE THREAD

**Issue by hanssens** [hanssens/extensions#41]
When using HttpFactory in a DI scenario, e.g. like in `services.AddHttpClient<SomeClient>()`, it may keep adding Accept headers to the request. Eventually resulting in 400 BadRequest error. Not sure if its limited to registering it Transient or Singleton, either way this should not occur.

**Comment by hanssens** [comment]
Example response:

```
{"error":"[BadRequest] Method: GET, RequestUri: 'https://<some-endpoint>', 
Version: 1.1, 
Content: System.Net.Http.StringContent, 
Headers:\r\n{\r\n  apikey: <some-api-key> \r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
Accept: application/json\r\n  
... etc.

Accept: application/json\r\n  
Request-Context: appId=cid-v1:e247526a-7e45-4e4f-8446-e470f0ec1d40\r\n  
Request-Id: |35e851730f03d94fae874d4f7204da3c.fe5192a33f62fd4c.\r\n  
traceparent: 00-35e851730f03d94fae874d4f7204da3c-fe5192a33f62fd4c-00\r\n  
Content-Type: application/json; charset=utf-8\r\n  
Content-Length: 0\r\n}"}
```

**Comment by hanssens** [comment]
Fixed in [release v2.2.0](https://github.com/hanssens/extensions/releases/tag/v2.2.0).
